dataviva/static/js/modules/basic_course.js

```diff
@@ -40,34 +40,34 @@ $(document).ready(function(){
 
     //INDICATORS ADD
     Indicator.add({
-        url: "http://api.staging.dataviva.info/sc/year?year=2014&" + filter,
-        title: dictionary['number_enrolled'],
+        url: "http://api.staging.dataviva.info/sc/year?year=2016&" + filter,
+        title: dictionary['number_enrolled'] + ['2016'],
```

Comment:
Fix the title from:
`title: dictionary['number_enrolled'] + ['2016'],`
to:
`title: dictionary['number_enrolled'] + ' (2016)',`